src/cacao.ts

```diff
@@ -42,15 +53,15 @@ export namespace Cacao {
       p: {
         domain: siweMessage.domain,
         iat: Date.parse(siweMessage.issuedAt),
-        iss: siweMessage.address,
+        iss: `did:pkh:eip155:1:${siweMessage.address}`,
         chainId: Number(siweMessage.chainId),
```

Comment:
`chainId` should not be part of the Cacao object, rather it should be included in the `iss`
```suggestion
        iss: `did:pkh:eip155:${Number(siweMessage.chainId)}:${siweMessage.address}`,
```